apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.js

```diff
@@ -0,0 +1,149 @@
+// @ts-check
+import * as express from 'express';
+const asyncHandler = require('express-async-handler');
+import * as sqldb from '@prairielearn/postgres';
+import * as error from '@prairielearn/error';
+import { getEnrollmentForUserInCourseInstance } from '../../models/enrollment';
+import { selectUserByUid } from '../../models/user';
+
+const router = express.Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+async function getUserIdAndCheckEnrollment({ uid, course_instance_id }) {
+  const user = await selectUserByUid(uid);
+  if (!user) {
+    throw error.make(400, `User ${uid} does not exist.`);
+  }
+
+  const enrollment = await getEnrollmentForUserInCourseInstance({
+    user_id: user.user_id,
+    course_instance_id: course_instance_id,
+  });
+  if (!enrollment) {
+    throw error.make(400, `User ${uid} is not enrolled in this course instance.`);
+  }
+
+  return user.user_id;
+}
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const result = await sqldb.queryAsync(sql.select_assessment_access_policies, {
+      assessment_id: res.locals.assessment.id,
+    });
+    res.locals.policies = result.rows;
+    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
```

Comment:
Similarly to the `GET` handler, we need permissions checks here. We should require "Student Data Editor" permissions.

Reply:
It looks like this was done in d485cf6?

Reply:
Yes, I was just waiting to confirm I checked the correct permissions, but I see that you resolved the earlier convo!